src/scripts/deck_machine_room.asm

```diff
@@ -138,14 +138,14 @@ Func_d96c:
 	ret
 
 ClubMapNames:
-	tx FightingClubMapNameText
-	tx RockClubMapNameText
-	tx WaterClubMapNameText
-	tx LightningClubMapNameText
-	tx GrassClubMapNameText
-	tx PsychicClubMapNameText
-	tx ScienceClubMapNameText
-	tx FireClubMapNameText
+	tx FightingClubMapName
+	tx RockClubMapName
+	tx WaterClubMapName
+	tx LightningClubMapName
+	tx GrassClubMapName
+	tx PsychicClubMapName
+	tx ScienceClubMapName
+	tx FireClubMapName
```

Comment:
I kind of liked the consistency of all text identifiers ending in the "Text" suffix. Thoughts?

Reply:
Well I guess we've already broken that convention for things like in data/cards.asm